src/extractor/guidance/sliproad_handler.cpp

```diff
@@ -698,24 +698,20 @@ bool SliproadHandler::isValidSliproadArea(const double max_area,
 }
 
 bool SliproadHandler::isValidSliproadLink(const IntersectionViewData &sliproad,
-                                          const IntersectionViewData &first,
+                                          const IntersectionViewData & /* first */,
                                           const IntersectionViewData &second) const
 {
-    // If the Sliproad is not a link we don't care
+    // If the sliproad is not a link we don't care
     const auto &sliproad_data = node_based_graph.GetEdgeData(sliproad.eid);
     if (!sliproad_data.road_classification.IsLinkClass())
     {
         return true;
     }
 
-    // otherwise neither the first road leading to the intersection we shortcut
-    const auto &first_road_data = node_based_graph.GetEdgeData(first.eid);
-    if (first_road_data.road_classification.IsLinkClass())
```

Comment:
Have you tried what happens, if you explicitly allow MotorwayLink classes here? Ramps are different than the primary/.. links, right?